Files/ViewModels/PreviewPaneViewModel.cs

```diff
@@ -30,8 +30,11 @@ public List<ListedItem> SelectedItems
             get => selectedItems;
             set
             {
-                SetProperty(ref selectedItems, value);
-                SelectedItem = SelectedItems?.FirstOrDefault();
+                if(value != selectedItems)
```

Comment:
```
if (SetProperty(ref selectedItems, value))
{
    SelectedItem = SelectedItems?.FirstOrDefault();
}
```

Reply:
@d2dyno1 Alright, I made the changes.
On a side note, the re-request a review button stopped working.